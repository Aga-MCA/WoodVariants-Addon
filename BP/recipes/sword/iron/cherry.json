{
  "format_version": "1.12",
  "minecraft:recipe_shaped": {
    "description": {
      "identifier": "iron_cherry_sword"
    },
    "tags": [
      "crafting_table",
      "aga"
    ],
    "pattern": [
      "P",
      "P",
      "S"
    ],
    "key": {
      "S": {
        "item": "aga_wood_variants:cherry_stick"
      },
      "P": {
        "item": "minecraft:iron_ingot"
      }
    },
    "result": {
      "item": "aga_wood_variants:iron_cherry_sword"
    }
  }
}