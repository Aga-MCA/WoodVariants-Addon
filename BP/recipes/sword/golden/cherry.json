{
  "format_version": "1.12",
  "minecraft:recipe_shaped": {
    "description": {
      "identifier": "golden_cherry_sword"
    },
    "tags": [
      "crafting_table",
      "aga"
    ],
    "pattern": [
      "P",
      "P",
      "S"
    ],
    "key": {
      "S": {
        "item": "aga_wood_variants:cherry_stick"
      },
      "P": {
        "item": "minecraft:gold_ingot"
      }
    },
    "result": {
      "item": "aga_wood_variants:golden_cherry_sword"
    }
  }
}